react-ui/src/constants/userTypes.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
A file was created to standardize the userTypes across the front end components. Unfortunately, this constants file cannot be in the root folder (for sharing across the server and react-ui codebase) due to limitations with React.

Reply:
Understood - this does make sense.